storage/mysql/storage_test.go

```diff
@@ -282,8 +282,12 @@ func TestMain(m *testing.M) {
 		glog.Errorf("MySQL not available, skipping all MySQL storage tests")
 		return
 	}
-	DB = openTestDBOrDie()
-	defer DB.Close()
-	cleanTestDB(DB)
-	os.Exit(m.Run())
+
+	var done func(context.Context)
+
+	DB, done = openTestDBOrDie()
+
+	status := m.Run()
+	done(context.TODO())
```

Comment:
Any reason not to use `context.Background()` here?

Reply:
I guess not, done.